Makefile

```diff
@@ -60,6 +60,22 @@ check: $(BIN)
 	(cd $(OUT); ../$(BIN) hello.elf)
 	(cd $(OUT); ../$(BIN) puzzle.elf)
 
+# variables for compliance
+COMPLIANCE_DIR ?= ./riscv-compliance
+export RISCV_PREFIX ?= riscv32-unknown-elf-
+export RISCV_TARGET = compliance-target
+export TARGETDIR = $(shell pwd)
+export XLEN = 32
+export JOBS ?= -j
+export WORK = $(TARGETDIR)/build/compliance
```

Comment:
Because the build process of compliance involves multiple makefiles in different places, I found out that the makefile from the compliance will fail to find our target file if I just set `TARGETDIR` to `.`. Therefore, I set it to the absolute path of our project root.